lib/src/components/NxBinaryDonutChart/types.ts

```diff
@@ -6,14 +6,18 @@
  */
 import { SVGAttributes } from 'react';
 import * as PropTypes from 'prop-types';
-import { requiredPercentNumber, optionalPercentNumber } from '../../util/customPropTypes';
+import { requiredValueNumber, optionalPercentNumber } from '../../util/customPropTypes';
 
 export interface Props extends SVGAttributes<SVGSVGElement> {
-  percent: number;
+  value: number;
```

Comment:
To not break backwards compatibility, `value` needs to be optional.  If you feel up to it you could try playing around with union types to create a type where at least one of `percent` or `value` are required.  See `NxModal`'s types for an example

Reply:
This is updated here b089e64cd333e786b70425b84c1f0e8972797b7e.